libpkgconf/pkg.c

```diff
@@ -406,6 +406,16 @@ pkgconf_pkg_new_from_file(pkgconf_client_t *client, const char *filename, FILE *
 	pkgconf_tuple_add(client, &pkg->vars, "pcfiledir", pc_filedir_value, true);
 	free(pc_filedir_value);
 
+        /* If pc_filedir is outside of sysroot_dir, clear pc_filedir
```

Comment:
Please fix the indentation to use actual tabs, thanks.

Reply:
Done